Pack-Smart/pack-smart-fe
# Refactor/quiz UI

## What's this PR do?
- Brings back universal scroll on nav logo/title
- Makes the last question optional
- Changes the order of questions, makes language around what is required
- Makes keydown on anything but a number on the days input impossible
- Reformats the multiple choice questions so they are aligned a little nicer
- Refactors functions with helper functions to break them down.

## Where should the reviewer start?
- Quiz.js
- quizDetails.js
- QuestionInput.js
- MultipleChoice.js

## How should this be manually tested?
- Fill out the entire quiz except the last question to make sure you can now proceed.
- Don't fill out at least one required field and to make sure you scroll back to the top with the error message.
- Just look at it!

## What are the relevant tickets?
- closes #15 
- closes #27 
- closes #29 
- closes #35

## Did you update the README?
- Nope
